service/display_feature.go

```diff
@@ -0,0 +1,44 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package service // import "go.opentelemetry.io/collector/service"
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"go.opentelemetry.io/collector/service/internal/graph"
+)
+
+func DisplayFeatures() error {
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
+	fmt.Fprintf(w, "ID\tEnabled\tStage\tDescription\n")
+	data := graph.GetFeaturesTableData()
+	for _, row := range data.Rows {
+		fmt.Fprintf(w, "%s\t%v\t%s\t%s\n",
+			row.ID,
+			row.Enabled,
+			row.Stage,
+			row.Description)
+	}
+	return w.Flush()
+}
+
+func DisplayFeature(id string) error {
+	data := graph.GetFeaturesTableData()
```

Comment:
Please call directly `VisitAll`, no need to depend on some other feature to implement this feature.